docs/source/pages/apps.rst

```diff
@@ -0,0 +1,53 @@
+.. _appis:
+
+Applications
```

Comment:
we will want to put a link the UKAEA software catelogue when the time comes for applications that we develop. We should probably put install instructions here?